e2e_test/source/basic_test.slt

```diff
@@ -13,7 +13,7 @@ statement ok
 drop source s
 
 statement ok
-create source s (v1 int not null, v2 char(8) not null) with ( 'upstream.source' = 'kafka', 'kafka.topic' = 'kafka_2_partition_topic', 'kafka.bootstrap.servers' = 'localhost:29092' ) row format 'json'
+create materialized source s (v1 int not null, v2 varchar not null) with ( 'connector' = 'kafka', 'kafka.topic' = 'kafka_2_partition_topic', 'kafka.brokers' = '127.0.0.1:29092' ) row format json
```

Comment:
```suggestion
create materialized source s (v1 int, v2 varchar) with ( 'connector' = 'kafka', 'kafka.topic' = 'kafka_2_partition_topic', 'kafka.brokers' = '127.0.0.1:29092' ) row format json
```

It's no use to specify `not null`, we always ignore it internally.

Reply:
fixed